MBTI Quiz/dist/index.html

```diff
@@ -44,357 +38,353 @@
 
   </nav>
 
-  <!-- Card -->
-
-  <!-- Card -->
-
-
-
   <section id = "top" class = "question1" >
-      <br><br><br><br>
-      <div class="card text-center container-small shadow-lg">
+    <br><br><br><br>
```

Comment:
Still seeing the unnecessary `<br>` Tags

Reply:
removed most of these but had to keep some as on the traits and careers and results cards even with margin-top on them when scrolling to those sections it would but off a section of the card without the <br>'s